tests/src/test/scala/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -164,11 +164,12 @@ abstract class ApiGwTests extends BaseApiGwTests {
     rr.stderr should include("The supplied authentication is invalid")
   }
 
-  def replaceStringInFile(fileName: String, regex: Regex, replacement: String): String = {
+  def replaceStringInFile(fileName: String, replacements: Map[Regex, String]): String = {
```

Comment:
note that the ApiGwTests.scala cli tests have move to the apache/incubator-openwhisk-cli repo.